client/components/Admin/ExportFeedback.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useEffect, useState } from 'react';
+import { VStack, Button, Box, Text, useToast, HStack, Stack, FormLabel, Select } from '@chakra-ui/react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import useMediaQuery from '../../hooks/useMediaQuery';
+import { apiInstance, dhruvaAPI } from '../../api/apiConfig';
+import { useQuery } from '@tanstack/react-query';
+import { listServices } from '../../api/serviceAPI';
+
+const ExportFeedback = () => {
+    const smallscreen = useMediaQuery("(max-width: 1080px)");
+    const [FromDate, setFromDate] = useState(null);
+    const [selectedServiceID, setSelectedServiceID] = useState(null);
+    const [ToDate, setToDate] = useState(null);
+    const toast = useToast();
+    const [FromUnix, setFromUnix] = useState<number>(null);
+    const [ToUnix, setToUnix] = useState<number>(null);
+    const { data: serviceslist } = useQuery(["services"], () => listServices());
+    useEffect(() => {
+        if (FromDate) {
+          setFromUnix(Math.round(FromDate.getTime() / 1000));
+        }
+      }, [FromDate]);
+    
+    useEffect(() => {
+        if (ToDate) {
+          setToUnix(Math.round(ToDate.getTime() / 1000));
+        }
+      }, [ToDate]);
+
+    const handleClick = () => {
+        if (FromUnix && ToUnix) 
+        {
+
+          if (ToUnix - FromUnix < 0) {
+            toast({
+              title: 'From Date Should be less than To Date',
+              status: 'warning',
+              duration: 2000,
+              isClosable: true,
+            });
+          } 
+          else 
+          {
+            if(ToUnix - FromUnix > 604800)
+            {
+                toast({
+                    title: 'Time Slot cannot be greater than one week',
+                    status: 'warning',
+                    duration: 2000,
+                    isClosable: true,
+                });
+            }
+            else
+            {
+            // const url = `${process.env.NEXT_PUBLIC_BACKEND_API_URL}/services/feedback/export?serviceId=${selectedServiceID}&fromDate=${FromUnix}&toDate=${ToUnix}`;
+            const url = `https://api.dhruva.co/services/feedback/export?serviceId=${selectedServiceID}&fromDate=${FromUnix}&toDate=${ToUnix}`;
```

Comment:
Can you please remove the hardcoded URL and use the env var?

Reply:
Yes sure